### ISSUE TITLE
Local mode client get io timeout error

### ISSUE THREAD

**Issue by sunxidong9999** [bougou/go-ipmi#30]
Hi, when I use local mode client in one of my devices, I get this error:

`
Device ID                 : 1
Device Revision           : 1
Firmware Revision         : 1.0
IPMI Version              : 2.0
Manufacturer ID           : 7154 (0x1bf2)
Manufacturer Name         : Unknown
Product ID                : 1 (0x0001)
Product Name              : 0x01
Device Available          : yes
Provides Device SDRs      : no
Additional Device Support :
    Chassis Device
    FRU Inventory Device
    SEL Device
    SDR Repo Device
    Sensor Device
Aux Firmware Rev Info     :
    0x00
    0x00
    0x00
    0x00
panic: GetSELEntry failed, err: openSendRequest failed, err: failed to read from syscall conn: i/o timeout

goroutine 1 [running]:
main.main()
`

Device info:
CPU: Intel(R) Atom(TM) x7835RE
Memory: 8GB

The function calling chain is:
`client.GetSELEntries(0) -->  c.GetSELInfo() --> c.Exchange(request, response) --> c.exchangeOpen(request, response) --> c.openSendRequest(request) --> open.SendCommand(c.openipmi.file, req) --> conn.Read(readMsgFunc)`

By the way, if I use lanplus mode to the this device, this error will not happen

**Comment by bougou** [comment]
@sunxidong9999 I made some adjustments on the timeout setting for local mode client. (from default 10s to 20s). 

And You can also use `client.WithTimeout` method to set a custom value.
Try new version v0.5.3 to see if it resolves the problem.

**Comment by sunxidong9999** [comment]
It works for me, the new method solved my issue, thank you very much.

There is still one issue, I don't know whether it's my Golang env problem.

when I used command `go mod tidy`, I got this info:

`
go: finding module for package github.com/bougou/go-ipmi
go: downloading github.com/bougou/go-ipmi v0.5.3
go: bmc imports
        github.com/bougou/go-ipmi: github.com/bougou/go-ipmi@v0.5.3: verifying module: checksum mismatch
        downloaded: h1:ghFUlvIBFrQEtBhwjlOOk0gWiEeUVnCWS0SGoNWTK80=
        sum.golang.org: h1:nSJzTC00hvr5S3HvCGCMR/sc+dCFAjdKEMdyeFB1qmY=

SECURITY ERROR
This download does NOT match the one reported by the checksum server.
The bits may have been replaced on the origin server, or an attacker may
have intercepted the download attempt.

For more information, see 'go help module-auth'.
`

What I did:
`
go clean -modcache
rm go.mod go.sum
go mod init bmc
go mod tidy
`
